learning_journal/templates/write.jinja2

```diff
@@ -0,0 +1,31 @@
+{% extends "layout.jinja2" %}
+{% block title %}Learning Journal New Entry Page{% endblock %}
+
+{% block header %}
+<header  class="intro-header" style="background-image: url('{{ request.static_path('learning_journal:static/img/about-bg.jpg') }}')">
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
+                <div class="site-heading">
+                    <h1>Scribble here.</h1>
+                    <hr class="small">
+                    <span class="subheading">Reflect, record. What got learned?</span>
+                </div>
+            </div>
+        </div>
+    </div>
+    </header>
+{% endblock %}
+{% block body %}
+    <!-- Main Content -->
+    <main class="container">
+        <div class="col-md-10">
+            <form method="POST">
+            <label>
+              <input name="title" class="input_post_title" type="text" placeholder="Hed" required>
+            </label>
```

Comment:
Not gonna ding you on this because it's not Python or Jinja, but why is this `input` field surrounded in a `label` tag?